website/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.1",
   "author": "HashiCorp",
   "devDependencies": {
-    "@hashicorp/platform-cli": "^1.2.0",
```

Comment:
Again here need to check if this breaking upgrade affects anything else.

Reply:
Seems to me that the breaking change just upgrade the eslint version. So we should be fine.

Reply:
Confirmed this upgrade is 'okay'. [See this comment.](https://github.com/hashicorp/vagrant/pull/13059#pullrequestreview-1252499799)